bin/index_db.sh

```diff
@@ -287,6 +291,21 @@ if [ "$data_only" == "true" ]; then
     echo "$CREATE_MODEL" >> create_models.job.sh
 else
   eval "$CREATE_MODEL"
+
+  if ["$train_llm" == "true"]; then
+    echo "Training LLM Weights"
```

Comment:
I think it would be better to keep as a separate script. Plus, @jodaiber is doing a lot of changes to index_db.sh, so we might have to talk about the best way of doing this a lot.